.github/workflows/auto-merge-dependabot-prs.yml

```diff
@@ -28,14 +28,12 @@ jobs:
     if: ${{ github.actor == 'dependabot[bot]' && github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success' }}
     steps:
       - name: Development Code
-        uses: actions/checkout@v5
-
-      ###### GET PR NUMBER
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
```

Comment:
@mogul @btylerburton Do we want these changes to the yml files, or should I scope just the PKCE/refresh token changes?